### PR TITLE
Ability to replace with file content

### PR DESCRIPTION
**Pull request by mehtaphysical**

### CODE REVIEW

**Comment by ckaznocha** [comment]
@mehtaphysical I love the idea but I have some concerns about the implementation trying to intuit if the user wants to use a file. Would you be OK with moving this to a new parameter? Maybe something like `replacement_files`?

**Comment by mehtaphysical** [comment]
For sure. i'll make that change

File: cmd/marathon-resource/behaviors/template.go [hunk 1]
```diff
@@ -30,3 +36,23 @@ func parsePayload(p Params, path string) (io.Reader, error) {
...
+func replaceFiles(metadata []Metadata, replacements map[string]string, path string) (map[string]string, error) {
+	for _, v := range metadata {
+		fileValue, err := ioutil.ReadFile(filepath.Join(path, v.Value))
+		if err != nil {
+			return replacements, fmt.Errorf("Error replaceing %s from replacement_files: %v", v.Name, err)
+		}
+		replacements[v.Name] = strings.TrimSpace(string(fileValue))
+	}
+
+	return replacements
```
Comment (ckaznocha) [inline cmd/marathon-resource/behaviors/template.go]: It's not compiling, this line needs to be `return replacements, nil`. Mind fixing it?

**Comment by mehtaphysical** [comment]
thanks for checking. sorry about that

**Comment by ckaznocha** [comment]
thanks @mehtaphysical! I squashed your commits and included them in #27, github just isn't showing your branch as merged because of the squash.

**Pull request closed without merging**